src/containers/OpenSeadragonViewer.js

```diff
@@ -1,7 +1,7 @@
 import { compose } from 'redux';
 import { connect } from 'react-redux';
-import { withTranslation } from 'react-i18next';
 import flatten from 'lodash/flatten';
+import { withTranslation } from 'react-i18next';
```

Comment:
I think this one didn't get deleted (here and below in `enhance`)

Reply:
OpenSeadragonViewer is still a class (until #4014), so we're stuck with the HOC.